frontend/src/forms/ParcelDetailForm.test.tsx

```diff
@@ -19,8 +19,10 @@ import BuildingForm from './subforms/BuildingForm';
 import EvaluationForm from './subforms/EvaluationForm';
 import LandForm from './subforms/LandForm';
 import PidPinForm from './subforms/PidPinForm';
+import { render, getByText, fireEvent, act, wait } from '@testing-library/react';
 
 Enzyme.configure({ adapter: new Adapter() });
+jest.mock('lodash/debounce', () => jest.fn(fn => fn));
```

Comment:
even with fake timers jest can't figure out how to trigger lodash debounce properly.